velox/exec/tests/HashJoinTest.cpp

```diff
@@ -586,6 +586,7 @@ class HashJoinBuilder {
     }
     auto queryCtx = std::make_shared<core::QueryCtx>(executor_);
     std::shared_ptr<TempDirectoryPath> spillDirectory;
+    std::unique_ptr<TestSpillUtils::TestScopedSpillPct> testScopedSpillPct;
```

Comment:
We could just local var here?
```
testSpillPct{0}
...
TestScopedSpillPct spillInjection{testSpillPct};
```

Reply:
Then the local var will be away from where it is told to be 100. This achieves the same effect, right?